src/ManageCourses.Domain/Models/McOrganisation.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GovUk.Education.ManageCourses.Domain.Models
+{
+    public class McOrganisation
```

Comment:
It's not immediately obvious to me what the "Mc" prefix means. Could this be made clearer somehow? Or could it be removed if it's not necessary?